src/platforms/javascript/common/configuration/integrations/default.mdx

```diff
@@ -148,3 +148,12 @@ Sentry.init({
   },
 });
 ```
+
+
+### Dedupe
```

Comment:
You've added this at the very end of the page, but that puts it under "Modifying System Integrations" rather than in the list of default integrations. Not sure if you intended this.

Reply:
Good catch! Fixed and merging now